docs/src/pages/guides/DB2OnCloudSetup.md

```diff
@@ -47,35 +45,88 @@ The administrator is `BLUADMIN` but you need to create a credential.
-        "jdbcurl": "jdbc:db2://dashdb-txn-***-************.services.dal.bluemix.net:50000/BLUDB",
-        "uri": "db2://bluadmin:bluadmin-password-very-secret@dashdb-txn-***-************.services.dal.bluemix.net:50000/BLUDB",
-        "db": "BLUDB",
-        "dsn": "DATABASE=BLUDB;HOSTNAME=dashdb-txn-***-************.services.dal.bluemix.net;PORT=50000;PROTOCOL=TCPIP;UID=bluadmin;PWD=bluadmin-password-very-secret;",
-        "username": "bluadmin",
-        "ssljdbcurl": "jdbc:db2://dashdb-txn-***-************.services.dal.bluemix.net:50001/BLUDB:sslConnection=true;"
+``` json
+{
+  "apikey": "******",
+  "connection": {
+    ...
+    "db2": {
+      "authentication": {
+        "method": "direct",
+        "password": "P1234",
+        "username": "username1"
+      },
+      "certificate": {
+        "certificate_base64": "secret=",
+        "name": "9e529eec-97df-4574-8839-12345"
+      },
+      "composed": [
+        "db2://username1:P1234@1-2-3-4.databases.appdomain.cloud:31366/bludb?authSource=admin&replicaSet=replset"
+      ],
+      "database": "bludb",
+      "host_ros": [
+        "1-2-3-4.databases.appdomain.cloud:32116"
+      ],
+      "hosts": [
+        {
+          "hostname": "1-2-3-4.databases.appdomain.cloud",
+          "port": 31366
+        }
+      ],
+      "jdbc_url": [
+        "jdbc:db2://1-2-3-4.databases.appdomain.cloud:31366/bludb:user=<userid>;password=<your_password>;sslConnection=true;"
+      ],
+      "path": "/bludb",
+      "query_options": {
+        "authSource": "admin",
+        "replicaSet": "replset"
+      },
+      "replica_set": "replset",
+      "scheme": "db2",
+      "type": "uri"
     }
-    ```
+  },
+  "iam_apikey_description": "Auto-generated for key username1-fc89-48e1-a1ce-569ca6c6e3d5",
+  "iam_apikey_name": "Service credentials-1",
+  "iam_role_crn": "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+  "iam_serviceid_crn": "secret",
+  "instance_administration_api": {
+    "deployment_id": "crn:v1:bluemix:public:dashdb-for-transactions:us-east:a/secret:",
+    "instance_id": "crn:v1:bluemix:public:dashdb-for-transactions:us-east:a/secret::",
+    "root": "https://api.us-east.db2.cloud.ibm.com/v4/ibm"
+  }
+}
+```
 
 4. Save these details for later, as these properties are needed to deploy and manage the IBM FHIR Server schema.
 
-For improved security, the **BLUADMIN** user should only be used to deploy the schema objects (tables, indexes, stored procedures etc) and administer the database, NOT for connecting from the fhir-server application.
+For improved security, the **admin** user should only be used to deploy the schema objects (tables, indexes, stored procedures etc) and administer the database, NOT for connecting from the IBM FHIR Server application.
+
+### **Adding the FHIRSERVER user**
+
+Following the least-privilege principle, the IBM FHIR server itself recommends running as db user, not db administrator.
+
+# Using Db2 Auth
```

Comment:
I wonder if we should just recommend one or the other.  Maybe always go with the IAM approach?

Reply:
I think offering both makes sense to me. Since I seem to be troubleshooting folks using both.